src/sql/foglamp_init_data.sql

```diff
@@ -73,7 +73,10 @@ INSERT INTO foglamp.configuration ( key, description, value )
 INSERT INTO foglamp.configuration ( key, description, value )
      VALUES ( 'SYPURGE', 'System Purge', to_jsonb( '{ "retention" : 259200, "last purge" : "' || now() || '" }' ) );
 
-
+-- COAP:  CoAP device server
+--        plugin: python module to load dynamically
+INSERT INTO foglamp.configuration ( key, description, value )
+     VALUES ( 'COAP', 'CoAP Plugin Configuration', ' { "plugin" : { "type" : "string", "value" : "foglamp.device.coap_device", "default" : "foglamp.device.coap_device", "description" : "Python module name of the plugin to load" } } ');
```

Comment:
Note than this is the fully specified module to import.

Reply:
I don't get why we have the fog lamp.device.coap_device in the config. T~his is meant to be dealt with programmatically when we do the import. This is a python and code organisation issue, not something to. burden the end user with, they just want to know they are using the coap plugin, where the python code lives is our problem not theirs.

Reply:
fixed